node_modules/oae-core/topnavigation/topnavigation.html

```diff
@@ -13,10 +13,10 @@
         <!-- RIGHT MENU -->
         <div id="topnavigation-right" class="col-xs-5 text-right">
             <!-- SEARCH FORM -->
-            <form action="." id="topnavigation-search-form" class="form-inline" role="search" aria-label="__MSG__TOPNAV_SEARCH_ARIA_LABEL__">
+            <form action="." id="topnavigation-search-form" class="form-inline" role="search">
                 <div class="form-group">
                     <label for="topnavigation-search-query" class="control-label sr-only">__MSG__SEARCH__</label>
-                    <input type="text" id="topnavigation-search-query" name="topnavigation-search-query" class="search-query form-control pull-left" title="__MSG__SEARCH_FOR_CONTENT_FOLDERS_PEOPLE_GROUPS__"/>
+                    <input type="text" id="topnavigation-search-query" name="topnavigation-search-query" class="search-query form-control pull-left" />
```

Comment:
Removing the title causes it to no longer have a tooltip. Given that the search field doesn't have a visible label or placeholder, wouldn't it be better to keep the title in place?